packages/eui/src/components/tabs/tabs.tsx

```diff
@@ -67,9 +75,41 @@ export const EuiTabs = forwardRef<EuiTabRef, EuiTabsProps>(
       bottomBorder && styles.bottomBorder,
     ];
 
+    const handleTabStop = (event: KeyboardEvent) => {
+      const currentRef = elementRef.current;
+      const tabs = currentRef?.querySelectorAll('[role="tab"]');
+      if (!tabs) return;
```

Comment:
```suggestion
      if (!tabs.length) return;
```

`querySelectorAll` will return an empty list/array which passes a truthy check, so this current early return won't work unless we check the length

Reply:
You're correct! I was focusing on safe-guarding against `elementRef.current` being `null` but didn't think of an empty array. I'll do:
```suggestion
      if (!tabs?.length) return;
```